### ISSUE TITLE
macOS Mojave & latest Xcode unable to build

### ISSUE THREAD

**Issue by Ralphaos** [snowflakedb/libsnowflakeclient#126]
The latest macOS/Xcode combo no longer supports i386. In order to compile for Mojave I needed to modify

cmake_opts+=("-DCMAKE_OSX_ARCHITECTURES=x86_64;i386") to be
cmake_opts+=("-DCMAKE_OSX_ARCHITECTURES=x86_64")

In build_libsnowflakeclient.sh.

**Comment by notkriswagner** [comment]
Yes, we still keep the `i386` architecture for support on older versions of macOS. I'll update the build script to skip the `i386` architecture for newer macOS builds.
